### PR TITLE
 #29 - Edit schedule

### PR DESCRIPTION
**Pull request by filfreire**
plus #30, replace names and talk names

related issues: #29 #30 

@FabioCB @filipemcarvalho kindly CR

Results:

speakers_tba
schedule

### CODE REVIEW

File: _data/sessions.yml [hunk 1]
```diff
@@ -1,59 +1,106 @@
...
+  description: "TBA"
   subtype: keynote
   speakers: [7]
   language: en
-  complexity: "Expert"
-  presentation: "https://speakerdeck.com/gdglviv/michal-tajchert-wearables-views"
-  video: "//www.youtube.com/embed/v3SCiUqI1yQ"
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
```
Comment (FabioCB) [inline _data/sessions.yml]: we can add videos and slides later.
Remove for now

File: _data/sessions.yml [hunk 2]
```diff
@@ -1,59 +1,106 @@
 -
   id: 307
-  title: "Coffee Break"
-  place: "Hall"
+  title: "Interval"
```
Comment (FabioCB) [inline _data/sessions.yml]: I think coffee break is better

File: _data/sessions.yml [hunk 3]
```diff
@@ -1,59 +1,106 @@
...
+  place: "TBA"
   service: true
+-
+  id: 309
+  title: "Arrival"
+  place: "TBA"
+  service: true
+-
+  id: 500
+  title: "Opening"
```
Comment (FabioCB) [inline _data/sessions.yml]: how about Keynote?

File: _data/schedule.yml [hunk 4]
```diff
@@ -1,50 +1,19 @@
...
+    - {startTime: "11:45",endTime: "",sessionIds: [307]}
+    - {startTime: "11:00",endTime: "",sessionIds: [002]}
+    - {startTime: "12:00",endTime: "",sessionIds: [003]}
+    - {startTime: "13:00",endTime: "",sessionIds: [308]}
+    - {startTime: "14:00",endTime: "",sessionIds: [004]}
+    - {startTime: "15:00",endTime: "",sessionIds: [005]}
+    - {startTime: "15:45",endTime: "",sessionIds: [307]}
+    - {startTime: "16:00",endTime: "",sessionIds: [006]}
+    - {startTime: "17:00",endTime: "",sessionIds: [007]}
+    - {startTime: "18:00",endTime: "",sessionIds: [501]}
```
Comment (filipemcarvalho) [inline _data/schedule.yml]: newline in the eof

File: _data/sessions.yml (hunk 2 above)
Comment (filipemcarvalho) [reply _data/sessions.yml]: Or only: 'Break'

File: _data/speakers.yml [hunk 5]
```diff
@@ -1,66 +1,84 @@
...
+  bio: "TBA"
   thumbnailUrl: user-200.png
-  rockstar: true
-  ribbon:
-    - {abbr: "GDG", title: "Google Developer Group", url: "https://developers.google.com/groups/chapter/110301321530950732281/"}
-  social: 
-    - {name: "google-plus", link: "https://google.com/+ValentynShybanov"}
-    - {name: "site", link: "http://olostan.name/"}
\ No newline at end of file
+  rockstar: false
```
Comment (filipemcarvalho) [inline _data/speakers.yml]: newline in the eof

File: _data/sessions.yml (hunk 1 above)
Comment (filfreire) [reply _data/sessions.yml]: done thanks :+1:

File: _data/sessions.yml (hunk 2 above)
Comment (filfreire) [reply _data/sessions.yml]: done thanks :+1:

File: _data/sessions.yml (hunk 3 above)
Comment (filfreire) [reply _data/sessions.yml]: done thanks :+1:

File: _data/schedule.yml (hunk 4 above)
Comment (filfreire) [reply _data/schedule.yml]: done thanks :+1:

File: _data/speakers.yml (hunk 5 above)
Comment (filfreire) [reply _data/speakers.yml]: done thanks :+1:

**Comment by filfreire** [comment]
@FabioCB @filipemcarvalho submitted improvs

**Pull request merged**